src/Parallel-Tests.ps1

```diff
@@ -0,0 +1,32 @@
+param(
+    [string[]] $assemblies,
+    [string] $testFilter,
+    [string] $outDir)
+
+$xunitRunner = $(Join-Path $PSScriptRoot 'packages\xunit.runner.console.2.1.0\tools\xunit.console.exe ')
```

Comment:
Totally overkill :) I think it should be easier to just have a if/else to choose the path when we will be cross platform. Maybe we will not use the same exec or the same version?

I like the idea to get the latest one, will see if I can come up with a quick solution